src/containers/DevSite.jsx

```diff
@@ -111,18 +111,27 @@ class Devsite extends Component {
     );
   }
 
+  findRouteCategory(meta, currRoute, homeRoute) {
+    const shortRoute = currRoute.slice(1);
+    for (let i = 0; i < meta.length; i += 1) {
```

Comment:
Nit: Why `i+=1` and not `i++` here?

Reply:
I personally don't have an issue with them, but our original lint rules disallowed them so I just stuck with it. 

https://stackoverflow.com/questions/7245691/are-increment-decrement-operators-considered-bad-form-in-javascript

A lot of people consider it "bad form", not sure it's backed by a solid argument, but people not liking it might be enough.